### ISSUE TITLE
WCAG 1.4.11 Color Contrast violation on hovered controls in Fluent Tester

### ISSUE THREAD

**Issue by ksiler** [microsoft/fluentui-react-native#745]
The hover background on some of the controls (button, context menu options, etc) do not meet [WCAG 1.4.11](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) requirements for color contrast. 

_Success Criterion 1.4.11 Non-text Contrast (Level AA): The visual presentation of the following have a contrast ratio of at least 3:1 against adjacent color(s)._

### Environment Information

- **Package version(s)**: (fill this out, include which package manager you're using)
- **OS version**: (fill this out if relevant)

- **Platform**:
  - [ ] iOS
    - **Xcode version**:
  - [ ] macOS
    - **Xcode version**:
  - [x] win32
  - [ ] windows
  - [ ] android

### Please provide a reproduction of the bug:

Open Fluent UI Tester and hover on one of the buttons. Inspect the color contrast of the hovered button with an adjacent color with AccessibilityInsights.

![image](https://user-images.githubusercontent.com/22876140/123469155-af838000-d5c0-11eb-87d5-f3cde7b574ef.png)

#### Actual behavior:

Color Contrast between hovered control and surrounding controls is 1.118.1

#### Expected behavior:

Color Contrast between hovered control and surrounding controls should be 3:1.

### Priorities and help requested:

Are you willing to submit a PR to fix? Yes

Requested priority: Low/Normal?

Products/applications affected: (if applicable)

**Comment by rurikoaraki** [comment]
@ksiler would adding a border around the button on hover satisfy this requirement? (as long as the border is at least 3:1 contrast ratio?)

**Comment by ksiler** [comment]
I think WCAG 1.4.11 becomes more crucial when considering controls where the only information used to identify it doesn't meet contrast requirements (like with icon buttons):


![image](https://user-images.githubusercontent.com/22876140/151399305-d98fa6ec-b9c0-46a0-a031-2a8b31c35adf.png)

This [github conversation](https://github.com/w3c/wcag/issues/849) also seems to agree that the contrast between the button's text and the background is all that's considered when checking for WCAG 1.4.11 (see example #3 on this page).

**Comment by rurikoaraki** [comment]
@ksiler Thanks for looking into this! If this is the case, I think I'll close out this issue. We can reopen it if it's still a problem.
